src/components/workspaces/Tools.tsx

```diff
@@ -2,25 +2,45 @@ import React from "react";
 import { Button, Box } from "@material-ui/core";
 
 import tools from "app/tools";
+import { Tool, Workspace } from "@types";
 
 import t from "components/i18n";
 
-type ToolProps = { workspaceUri: string };
+type ToolButtonProps = {
+  tool: Tool;
+  workspace: Workspace;
+};
+
+const ToolButton: React.FC<ToolButtonProps> = ({ tool, workspace }) => {
+  if (workspace.vocabularies.length > 0) {
+    return (
+      <Button
```

Comment:
Would not it be enough to have a single Button with disabled={workspace.vocabularies.length > 0} ?

Reply:
It would not, because of UX. The button looks like a button, but in fact it's an anchor with the `href` atribute. So you can see the URL in the status bar on hover, you can right click and copy URL, you can right click and open in a new tab... However, the anchor masked as a button cannot be disabled. So either a disabled button is rendered, or an anchor that looks like a button.